templates/observability/managed.html

```diff
@@ -0,0 +1,629 @@
+{% extends "observability/base_observability.html" %}
+
+{% block title %}Managed Open Source Observability{% endblock %}
+
+{% block meta_description %}Managed open-source observability: how to achieve end-to-end cloud observability while maintaining your business focus and without spending your budget keeping the lights on.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1N11otwvy2TWoK4j9e8iYJLNLYQyWbIPL82GVkm8WHhE/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Managed Open Source Observability</h1>
+      <h2 class="p-heading--four">You focus on your business, we run the monitoring tools for you</h2>
+      <p>Uptime for monitoring tools is quintessential for your business. Without reliable monitoring tools, you cannot know what else is not working right.</p>
+      <p>81% of CIOs are planning to increase their observability budgets, you could rather use yours more efficiently.</p>
+      <p>Focus on driving your unique business, and we will run the best-in-class open source monitoring tools you need for the observability of your applications.</p>
+      <p>
+        <a class="p-button--positive">Talk with an expert</a>
+      </p>
+      <p>
+        <a href="/engage/managed-services-overcoming-cio-challenges-2021">Learn how to focus on your unique business with managed IT services&nbsp;&rsaquo;</a>
```

Comment:
In the copy, this has not now become a button CtA with label "Read the CIO guide to managed services"